dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/organisationunit/hibernate/HibernateOrganisationUnitGroupStore.java

```diff
@@ -57,6 +57,12 @@ public HibernateOrganisationUnitGroupStore( SessionFactory sessionFactory, JdbcT
     @Override
     public List<OrganisationUnitGroup> getOrganisationUnitGroupsWithGroupSets()
     {
-        return getQuery( "from OrganisationUnitGroup o where o.groupSet is not null" ).list();
```

Comment:
**Note:** This was a bug, `o.groupSet` does not exist, the operations fails when used, but this wasn't detected by the test because the particular method for this lacked the `@Test` annotation.

Reply:
Maybe we can backport this individual bug-fix?